src/utilities/core-utils.ts

```diff
@@ -129,27 +225,41 @@ const _timer = (name: string) => {
     };
 };
 
+
+/**
+ * Invokes the iteratee function n times, returning an array of the results of each invocation. The iteratee
+ * is invoked with one argument; (index).
+ *
+ * @param n The number of times to invoke iteratee.
+ * @param iteratee The function invoked per iteration.
+ * @return Returns the array of results.
+ */
+const times = <TResult>(
```

Comment:
Oh, sweet! Didn't even realize this was something we had available. Kinda the same idea that I had with a recent function export from the testing package to loop a test body a certain number of times.